workbench-core/environments/src/postDeployment/cognitoSetup.test.ts

```diff
@@ -96,30 +63,7 @@ describe('CognitoSetup', () => {
       jest.spyOn(cognitoSetup, 'adminCreateUser').mockImplementation();
       const returnVal = await cognitoSetup.run();
       expect(returnVal).toBeUndefined();
-    });
-
-    test('run: Groups already exist, user already exists, add user to Admin group', async () => {
-      const cognitoSetup = new CognitoSetup(constants);
-      const cognitoMock = mockClient(CognitoIdentityProviderClient);
-      mockCognito(cognitoMock);
-
-      const responseMetadata = { httpStatusCode: 400 };
-      cognitoMock.on(AdminCreateUserCommand).rejects(
-        new GroupExistsException({
-          $metadata: responseMetadata,
-          message: ''
-        })
-      );
-      cognitoMock.on(AdminCreateUserCommand).rejects(
-        new UsernameExistsException({
-          $metadata: responseMetadata,
-          message: ''
-        })
-      );
-      jest.spyOn(cognitoSetup, 'createGroup').mockImplementation();
-      jest.spyOn(cognitoSetup, 'adminCreateUser').mockImplementation();
-      const returnVal = await cognitoSetup.run();
-      expect(returnVal).toBeUndefined();
+      expect(cognitoSetup.adminCreateUser).toBeCalledTimes(1);
```

Comment:
It looks like we're expecting the same outputs as the above test even though the user already exists. What are we verifying in this scenario?

Reply:
the response from `AdminCreateUserCommand` has been changed from a `resolve` to a `reject`. this is running the error path.